test/system/libs/helpers.bash

```diff
@@ -258,14 +257,13 @@ function pull_distro_image() {
   # No need to copy if the image is already available in Podman
   run $PODMAN image exists ${image}
   if [[ "$status" -eq 0 ]]; then
-    return
+    return 0
```

Comment:
This isn't about this specific change, but a more general comment.

Why are we running `$PODMAN image exists` under Bats' `run`?  It's meant for cases where we are want to *test* a command.  Here, we are only doing a boolean check, so we don't need all the overhead of `run`.  Can we do something like this:
```
if "$PODMAN" image exists "$image" >/dev/null 2>&1; then
    return 0
fi
```